### PR TITLE
Fix `pgcli` just commands for the api and catalog

### PR DESCRIPTION
**Pull request by krysal**
## Fixes
The command is not working on `main`.

## Description

This PR completes the default credentials to connect to the database seamlessly.

## Testing Instructions

Just try it here and compare with the `main` branch.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: api/justfile [hunk 1]
```diff
@@ -64,9 +64,9 @@ _curl-get path host="localhost:50280":
 stats media="images" host="localhost:50280":
     just _curl-get "{{ media }}/stats/" {{ host }}
 
-# Launch a `pgcli` shell in the web container (requires typing credentials)
-pgcli db_host="db" db_name="openledger" db_user="deploy":
-    env DC_USER="opener" just ../exec web pgcli -h {{ db_host }} openledger
+# Launch a `pgcli` shell in the web container
+pgcli db_host="db" db_name="openledger" db_user="deploy" db_password="deploy" db_port="5432":
+    env DC_USER="opener" just ../exec web pgcli postgresql://{{ db_user }}:{{ db_password }}@{{ db_host }}:{{ db_port }}/{{ db_name }}
```
Comment (AetherUnbound) [inline api/justfile:69]: That's right!! I forgot we could use the connection URI, that's so much easier

File: api/justfile [hunk 2]
```diff
@@ -64,9 +64,9 @@ _curl-get path host="localhost:50280":
 stats media="images" host="localhost:50280":
     just _curl-get "{{ media }}/stats/" {{ host }}
 
-# Launch a `pgcli` shell in the web container (requires typing credentials)
-pgcli db_host="db" db_name="openledger" db_user="deploy":
-    env DC_USER="opener" just ../exec web pgcli -h {{ db_host }} openledger
+# Launch a `pgcli` shell in the web container
+pgcli db_host="db" db_name="openledger" db_user="deploy" db_password="deploy" db_port="5432":
```
Comment (AetherUnbound) [inline api/justfile:68]: It looks like this requires `just up` first, can/should we add that as a dependency?

```suggestion
pgcli db_host="db" db_name="openledger" db_user="deploy" db_password="deploy" db_port="5432": up
```

**Review by AetherUnbound — APPROVED** [review]
Works great for me! One non-blocking suggestion.

File: api/justfile (hunk 2 above)
Comment (dhruvkb) [inline api/justfile:68]: I would recommend moving the port to be located next to the host as that makes the order easier to memorize and keeps the params that are likely to change closer to the start.

```suggestion
pgcli db_host="db" db_port="5432" db_name="openledger" db_user="deploy" db_password="deploy":
```

**Review by dhruvkb — APPROVED** [review]
Nice! The `pgcli` recipe for the catalog could use this improvement too!

**Comment by krysal** [comment]
Thank you both for the suggestions! I fixed the just command of the catalog too 😄

**Pull request merged**